Maestro/subscriptions.json

```diff
@@ -405,6 +426,27 @@
             "/verbosity:Normal"
           ]
         },
+        // This handler will bring the Latest CoreCLR release/1.1.0 build into CoreCLR release/1.1.0
```

Comment:
This is a true CoreCLR 1.1.0 to CoreCLR 1.1.0 subscription--all it does is change [`CoreClrPackageVersion`](https://github.com/dotnet/coreclr/blob/6bc804a27da5a646c3a1c339e5b653503b379681/dependencies.props#L15) which can optionally be used to run tests.

We decided in the 1.1.0 thread that we'd keep the TFS dependencies at 1.0.0, so there's no coreclr 1.1.0 subscription on projectk-tfs.